packages/mgt/src/components/mgt-login/mgt-login.scss

```diff
@@ -117,3 +117,12 @@ mgt-login .popup-command {
     color: var(--theme-dark-color);
   }
 }
+
+:host(.rtl) .login-button {
```

Comment:
instead of class, have you tried setting an attribute on the shadow dom root and use that instead? You should be able to do selectors like

```css
.login-button[dir="rtl"] {

}
```

Reply:
I was trying to avoid:
![ignore](https://user-images.githubusercontent.com/11411686/96320583-9fde8100-0fc7-11eb-933d-ee7c26f6f19a.png)
Although it seems like with the latest implementation this is avoided anyways.